packages/generators/add-generator.ts

```diff
@@ -395,15 +395,11 @@ export default class AddGenerator extends Generator {
 								(p: boolean): void => {
 									if (p) {
 										this.dependencies.push(answerToAction.actionAnswer);
-										const normalizePluginName = answerToAction.actionAnswer.replace(
-											"-webpack-plugin",
-											"Plugin"
-										);
-										const pluginName = replaceAt(
-											normalizePluginName,
-											0,
-											normalizePluginName.charAt(0).toUpperCase()
-										);
+										let myPluginNameArray = answerToAction.actionAnswer.split("-")
```

Comment:
Could you extract this new logic into a separate function and test it against few different plugins? There are the native ones too `webpack.DefinePlugin`.

Reply:
+1, this could quickly turn into a mess

Reply:
^ What I mean by that is that like @ematipico is saying, is that it might be hard to verify that this works unless we test it thoroughly. A better thing to do, would be to abstract the logic so we can run unit tests on it and make sure that it is working as expected.

Reply:
Okay, So Should I add it in `utils` and write tests for it. If it's fine? @evenstensberg @ematipico

Reply:
Probably nice idea, or to add it within the generator itself if it's only used here

Reply:
@evenstensberg  Yeah....I was talking about the `utils` folder inside the `generators` package. Will add it there with test then

Reply:
That's a great location. Go for it

Reply:
okay 👍